enhancements/insights/rapid-recommendations.md

```diff
@@ -0,0 +1,350 @@
+---
+title: rapid-recommendations
+authors:
+  - "@jholecek-rh"
+  - "@tremes"
+reviewers: 
+  - "@deads2k"
+approvers: 
+  - "@deads2k"
+api-approvers: 
+  - None
+creation-date: 2024-01-03
+last-updated: 2024-02-22
+tracking-link: 
+  - https://issues.redhat.com/browse/CCXDEV-12213
+  - https://issues.redhat.com/browse/CCXDEV-12285
+see-also:
+  - "/enhancements/insights/conditional-data-gathering.md"
+replaces:
+  - None
+superseded-by:
+  - "/enhancements/our-past-effort.md"
+---
+
+# Insights Rapid Recommendations
+
+This enhancement proposal introduces a new approach to how the data collected by the Insights operator can be defined remotely.
+
+
+## Summary
+
+The Insights Operator collects various data and resources from the OpenShift and Kubernetes APIs. The definition of the collected data is mostly hardcoded in the operator's source code and largely locked in the corresponding OCP version.
```

Comment:
minor: please break on every sentence or about 100ish characters so it's easier to comment.